OpenTaiko/src/Common/ImGuiDebugWindow.cs

```diff
@@ -366,17 +364,126 @@ private static void Textures() {
+										"Tower Full Combo", "TEXTURE_LUA_TOWERFC");
+									currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.Tower_TopReached_PerfectScript,
+										"Tower Perfect Combo", "TEXTURE_LUA_TOWERPFC");
+									break;
+								case Difficulty.Dan:
+									currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.Dan_FailScript,
+										"Dan Clear Failed", "TEXTURE_LUA_DANFAILED");
+									currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.Dan_Red_PassScript,
+										"Dan Red Clear", "TEXTURE_LUA_DANCLEAR");
+									currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.Dan_Red_FullComboScript,
+										"Dan Red Full Combo", "TEXTURE_LUA_DANFC");
+									currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.Dan_Red_PerfectScript,
+										"Dan Red Perfect", "TEXTURE_LUA_DANPFC");
+									currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.Dan_Red_PassScript,
+										"Dan Gold Clear", "TEXTURE_LUA_DANGOLDCLEAR");
+									currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.Dan_Red_FullComboScript,
+										"Dan Gold Full Combo", "TEXTURE_LUA_DANGOLDFC");
+									currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.Dan_Red_PerfectScript,
+										"Dan Gold Perfect", "TEXTURE_LUA_DANGOLDPFC");
+									break;
+								default:
+									if (OpenTaiko.ConfigIni.bAIBattleMode) {
+										currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.AILoseScript,
+											"AI Clear Failed", "TEXTURE_LUA_AIFAILED");
+										currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.AIWinScript,
+											"AI Cleared", "TEXTURE_LUA_AICLEAR");
+										currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.AIWin_FullComboScript,
+											"AI Full Combo", "TEXTURE_LUA_AIFC");
+										currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.AIWin_PerfectScript,
+											"AI Perfect Combo", "TEXTURE_LUA_AIPFC");
+									}
+									else {
+										currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.FailedScript,
+											"Clear Failed", "TEXTURE_LUA_GAMEFAILED");
+										currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.ClearScript,
+											"Cleared", "TEXTURE_LUA_GAMECLEAR");
+										currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.FullComboScript,
+											"Full Combo", "TEXTURE_LUA_GAMEFC");
+										currentStageMemoryUsage += CTextureListPopup(OpenTaiko.stage演奏ドラム画面.actEnd.PerfectComboScript,
+											"Perfect Combo", "TEXTURE_LUA_GAMEPFC");
+									}
+									break;
+							}
+							#endregion
+							ImGui.TreePop();
+						}
+						else {
+							// Summarize mem usage from scripts even when the tab is closed
```

Comment:
Could this be refactored into a own method instead of adding this comment as a description?
The method would be shorter + we could write documentation.